backend/pages/models.py

```
@@ -71,7 +72,7 @@ def process_and_store_csv(self):
                     processed_data_frame = df_processing
 
                 columns = processed_data_frame.columns.tolist()
-                data_json = convert_df_to_json(processed_data_frame)
+                data_json = convert_df_to_json(processed_data_frame, precision=self.quarter.float_precision)
```

Pylint (3.3.7)

[convention] Line too long (108/100)

(C0301)